overlord/snapstate/snapmgr.go

```diff
@@ -390,20 +390,22 @@ func refreshScheduleNoWeekdays(rs []*timeutil.Schedule) error {
 	return nil
 }
 
-func (m *SnapManager) checkRefreshSchedule() ([]*timeutil.Schedule, error) {
+func (m *SnapManager) refreshSchedule() ([]*timeutil.Schedule, string, error) {
```

Comment:
Can you please add variable names to the return tuple. At this stage it's a bit unclear what the string would do (without reading the rest of the patch).